hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxySupport.java

```diff
@@ -373,22 +373,26 @@ private Data readBackupDataOrNull(Data key) {
     }
 
     protected Data putInternal(Data key, Data value, long ttl, TimeUnit timeunit) {
-        MapOperation operation = operationProvider.createPutOperation(name, key, value, getTimeInMillis(ttl, timeunit));
+        long timeInMillis = getTimeInMillis(ttl, timeunit);
```

Comment:
I pulled out the `getTimeInMillis(ttl, timeunit)` calls because in the follow-up PR these lines will be longer than 130 chars.